airflow/providers/amazon/aws/hooks/redshift_data.py

```diff
@@ -95,6 +100,12 @@ def execute_query(
 
         statement_id = resp["Id"]
 
+        if kwargs["ClusterIdentifier"] and kwargs["WorkgroupName"]:
```

Comment:
```suggestion
        if cluster_identifier and workgroup_name:
```

Reply:
Another solution: merge the two if into one with:

```python
if bool(cluster_identifier) == bool(workgroup_name):
```

Reply:
If we opt for this solution, we'll need to update the log message as well and `bool(...) is bool(...)` might be a better solution?

Reply:
Thank you both, addressed in 56cc6d8ea13031ca610250735c8b0071065b233f